00-HTML-CSS-basics/HTMLExercises/js/validate.js

```diff
@@ -0,0 +1,88 @@
+document.querySelector('#submit').addEventListener('click', function(e){
```

Comment:
Although this is nice, the idea is to implement a form using all pure HTML5 tag without javascript. Please redo.